src/lang/en-US.ts

```diff
@@ -423,27 +423,27 @@ const Collections: ResolvedIntlConfig['messages'] = {
     'collectionsPreview.tooManyBytesAndNoDocuments.message': `We reached the limit of how much data a web browser can comfortably read, and didn't find even reach the end of one document! This probably means that your documents are huge.`,
 };
 
-const entityCreateHeader = `Endpoint Config`;
+const endpointConfigHeader = `Endpoint Config`;
```

Comment:
I would like to have a content audit this quarter to remove duplication of entity workflow content and supply names that make it easy for anyone to update this file. Going to create an issue for this work once I finish reviewing this PR.